src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1462,9 +1462,9 @@ internal virtual IntermediateNode CreateIRCallNode(CallNode node, IRTranslatorCo
         /// By default, function does not attach any pre-processing for arguments.
         /// </summary>
         /// <param name="index">0 based index of argument.</param>
-        /// <param name="node">The original call node.</param>
+        /// <param name="argCount">The count of argument to call the function.</param>
```

Comment:
>The count of argument to call the function. [](http://example.com/codeflow?start=35&length=43)

Can you give an example of this parameter? It's not clear based on the description.

Reply:
Added more description.